person.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Person
+attr_reader :age, :phase
+attr_accessor :name
+
+  def initialize(options={})
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+    @phase = options.fetch(:phase, self.class::DEFAULT_PHASE)
```

Comment:
is this value coming in through the params or being set in the class? Which one?